src/Middlewares/Middleware.php

```diff
@@ -76,14 +79,19 @@ public function before(Request $request)
         }
 
         if (!$this->searchService->aliveTest()) {
-            $this->getLoggerObject()->error('Findologic search is not available!');
-
+            $this->getLoggerObject()->alert(
```

Comment:
So we had some internal discussion about this and decided to actually get rid of this log line.

Reason behind that is that we do not add such a log line for other plugins as well. Also a log line should inform someone to take action. In that case the customer can not take any action, as this could be a one time accident and if Findologic was actually down, the customer would notice that anyway and inform us via the Plenty forum or via our support.